README.tr.md

```diff
@@ -0,0 +1,85 @@
+# sway
+
+[English][en] - [日本語][ja] - [Français][fr] - [Українська][uk] - [Español][es] - [Polski][pl] - [中文-简体][zh-CN] - [Deutsch][de] - [Nederlands][nl] - [Русский][ru] - [中文-繁體][zh-TW] - [Português][pt] - [Danish][dk] - [한국어][ko] - [Română][ro] - **[Türkçe][tr]**
```

Comment:
Please remove the language selectors from the translated README, otherwise we need to update many files when adding a single translation.